### PR TITLE
feat: added product lisiting page

### PR DESCRIPTION
**Pull request by Prashantrajput11**
Added Product listing page:
it includes : Sidebar and Products showcase
image

### CODE REVIEW

**Comment by poornima-alugubelly** [comment]
I like the clean structure and naming good to merge 👍

**Review by poornima-alugubelly — APPROVED** [review]

File: css/homepage.css [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+.hero-primary-cta {
+  font-size: 2rem;
+  background: var(--primary-color);
+  color: var(--secondary-color);
+  width: 10%;
+}
+
+.hero-primary-cta:hover{
+	 background: #4f0e28;
```
Comment (MarkVed17) [inline css/homepage.css:92]: just some code-formatting here

File: css/homepage.css [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+.section-featured {
+  background-color: #f3e8ec;
+}
+
+.featured-wrapper {
+  height: auto;
+}
+
+.vt--card-container{
+	background: var(--black-s3);
```
Comment (MarkVed17) [inline css/homepage.css:108]: here too

File: pages/products.html [hunk 3]
```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+	<meta charset="utf-8" />
+	<title></title>
```
Comment (MarkVed17) [inline pages/products.html]: Add any title here.

**Review by MarkVed17 — APPROVED** [review]

File: pages/products.html (hunk 3 above)
Comment (Prashantrajput11) [reply pages/products.html]: fixed

File: pages/products.html [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+	<link rel="stylesheet" href="../shared/typography.css" />
+	<link rel="stylesheet" href="../shared/spacing-design.css" />
+	<link rel="stylesheet" href="../shared/layout.css" />
+	<link rel="stylesheet" href="../css/productslist.css" />
+</head>
+
+<body>
+	<div class="nav-container">
+		<div class="logo"><a href="/index.html" class="tx-bold">BreeZecart</a></div>
+
```
Comment (khushijohri2001) [inline pages/products.html:22]: You can remove extra spaces here.

**Review by khushijohri2001 — COMMENTED** [review]
I liked the naming part. The code looks good to me

**Pull request merged**